### PR TITLE
ACM-5191 Console-crashes-templates-imetadata-name

### PR DESCRIPTION
**Pull request by jeswanke**
Signed-off-by: John Swanke <jswanke@redhat.com>

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -10,8 +10,10 @@
...
 #webpack-dev-server-client-overlay,
 #react-refresh-overlay {
-  overflow: scroll;
-  height: 20% !important;
-  top: 80% !important;
+  height: 75px !important;
+  min-height: 75px !important;
+  left: 50% !important;
+  top: 0px !important;
+  width: 400px !important;
```
Comment (KevinFCormier) [inline frontend/src/App.css:17]: Did you mean to include this?

Comment (jeswanke) [reply frontend/src/App.css:17]: oop

**Review by KevinFCormier — APPROVED** [review]

**Review by KevinFCormier — CHANGES_REQUESTED** [review]
There are a bunch of untranslated strings hiding in here. Will open an issue to deal with those later, but for now, just asking for a slight adjustment to the wording.

**Pull request merged**